jabel-javac-plugin/build.gradle.kts

```diff
@@ -0,0 +1,123 @@
+plugins {
+    java
+    `maven-publish`
+    signing
+}
+
+val jabelVersion: String by project
+version = jabelVersion
+
+dependencies {
+    implementation(platform("net.bytebuddy:byte-buddy-parent:1.14.9"))
+    implementation("net.bytebuddy:byte-buddy")
+    implementation("net.bytebuddy:byte-buddy-agent")
+    implementation("net.java.dev.jna:jna:5.13.0")
+}
+
+// <editor-fold desc="Publishing and Signing">
+
+tasks.withType<Javadoc>{
+    options.source = "8"
+}
+
+java {
+    sourceCompatibility = JavaVersion.VERSION_1_8
+    targetCompatibility = sourceCompatibility
+    withJavadocJar()
+    withSourcesJar()
+}
+
+publishing {
+    publications {
+        create<MavenPublication>("mavenJava") {
+            artifactId = pomArtifactId
+            from(components["java"])
+            pom {
+                name.set(pomName)
+                packaging = pomPackaging
+                description.set(pomDescription)
+                url.set("https://github.com/pkware/jabel")
+                setPkwareOrganization()
+
+                developers {
+                    developer {
+                        id.set("all")
+                        name.set("PKWARE, Inc.")
+                    }
+                }
+
+                scm {
+                    connection.set("scm:git:git://github.com/pkware/jabel.git")
+                    developerConnection.set("scm:git:ssh://github.com/pkware/jabel.git")
+                    url.set("https://github.com/pkware/jabel")
+                }
+
+                licenses {
+                    license {
+                        name.set("Apache License, Version 2.0")
```

Comment:
Double check that this license is the same as the original repository

Reply:
Aye it is, see lines 9-14 in the deleted gradle/publishing.gradle file